plugins/woocommerce-blocks/tests/e2e/tests/templates/checkout-template.block_theme.spec.ts

```diff
@@ -65,6 +64,7 @@ test.describe( 'Test the checkout template', () => {
 		await frontendUtils.addToCart();
 		await admin.page.goto( permalink );
 		await admin.page.locator( '#wp-admin-bar-site-editor a' ).click();
+		await admin.page.getByRole( 'button', { name: 'Get started' } ).click();
```

Comment:
Is this for closing the modal? If someone is running this test locally multiple times (for debugging) would this line fail if the modal has already been dismissed in a previous run? Would it be better to check its visibility first?

Reply:
The non-deterministic state of the modal would be a side effect. Since [we reset DB for each test](https://github.com/woocommerce/woocommerce/pull/46125), side effects are no longer happening in blocks e2e tests. TL;DR we don't need to check if the modal is visible! 😄 🙌

Reply:
> Is this for closing the modal?

Yes, exactly. I added a comment in 6ee4755168af0acf9f13c63ef7487929d5437c94 to clarify a bit more what this code is doing.

> If someone is running this test locally multiple times (for debugging) would this line fail if the modal has already been dismissed in a previous run? Would it be better to check its visibility first?

My understanding is that since we are resetting the state between tests, it shouldn't make a difference, no? I made it conditional just in case, though.

Reply:
For some reason I missed @WunderBart's comment above before publishing my reply! :sweat: 

So based on it, I removed the conditional in c4aedc0a05ecb0cd5c137519fa68e526e52aa32d.

Reply:
Sounds good! Thanks folks :)